packages/angular/projects/dev/src/app/datagrid/conditional-selection/conditional-selection.html

```diff
@@ -6,6 +6,46 @@
 
 <h2>Datagrid: Conditional selection</h2>
 
+<h3>Lock the how table</h3>
+
+<button class="btn btn-primary" (click)="locked = !locked || false">Lock Table</button>
+
+<clr-datagrid [(clrDgSingleSelected)]="selected">
+  <clr-dg-column>User ID</clr-dg-column>
+  <clr-dg-column>Name</clr-dg-column>
+  <clr-dg-column>Creation date</clr-dg-column>
+  <clr-dg-column>Favorite color</clr-dg-column>
+
+  <clr-dg-row [clrDgSelectable]="locked" *clrDgItems="let user of users" [clrDgItem]="user">
```

Comment:
nit: Shouldn't we have `!locked` here? As I mentioned above, `locked` should represent the opposite of `selectable`.

Reply:
This is copy paste of the Stackblitz demo design to test the different order of arguments in the feature this demo could be removed or not - it has no effect of change...

I could rename it from `locked` to `state` and pass `false` or `true`

Reply:
No, `state` would make it much vaguer. I think `locked` is just fine. I was pointing out that if `locked` is for `clrDgSelectable `, it should represent the opposite value of itself. Right now, it looks like both `locked` and `clrDgSelectable` are using the same exact value.

Reply:
The value of `locked` is fine, it's controlled from the button above and it's designed to be able to lock and unlock the rows by pressing it (the button) for demo purpose. No need to invert it or change it.